homeassistant/components/nut/sensor.py

```diff
@@ -64,6 +89,8 @@ async def async_setup_entry(
 class NUTSensor(CoordinatorEntity, SensorEntity):
     """Representation of a sensor entity for NUT status values."""
 
+    coordinator: DataUpdateCoordinator[dict[str, str]]
```

Comment:
The coordinator entity is generic so we can type it like this nowadays.

https://github.com/home-assistant/core/blob/830cc278d3a4decb2dd9305305da281ac72282ed/homeassistant/components/twentemilieu/entity.py#L20-L29